paper/paper.tex

```diff
@@ -0,0 +1,946 @@
+
+This close relationship between measures and \emph{probability} measures is illustrated by the Lebesgue measure $\lambda$ on $X=\RR$. The measure of the full space is $\lambda(\RR) = \infty$. But restricting to the unit interval gives $\lambda([0,1])=1$. This \emph{restricted} measure is a probability distribution---the uniform distribution $\uU([0,1])$ describing the law of Julia function \verb|rand()| giving random numbers in the interval $[0,1]$.
+
+We have not yet stated \emph{for which} sets Kolmogorov's axioms have to hold for something to be called a measure or a probability:
+
+\subsection*{$\sigma$-algebras and how to like them}
+
+%As stated at the start of the previous section, 
+A (probability) measure on $X$ need not assign a (probability) mass to every possible subset $A\subset X$,
+but only to each \emph{event} for a probability measure, or \emph{measurable set} more generally.
+Statements we make about sets should be understood as restricted to this set-of-sets, which can be different from one measure to the next.
+
+Now, Kolmogorov's axioms require a given measure to be defined for $X$, and they also require that this set of measurable sets is closed under the operations which the axioms allow (complements and unions of sets or sequences of sets). This imposes an algebraic structure to the measurable sets, called a \emph{$\sigma$-algebra}, which is exactly that: A set of sets $\subset X$ (or $\subset \Omega$ in particular) closed under complementation and countable unions and intersections.
+
+This raises the question: Why not just use the \href{https://en.wikipedia.org/wiki/Power_set}{\emph{power set}}, the ``set of all sets'', which is, after thinking about it, such a $\sigma$-algebra?
+
+A key observation is that having defined probability mass for a number of relatively simple sets, for example on all intervals,
+Kolmogorov's axioms give a recipe to compute the probability mass of more complicated sets, and with those, even more in new applications of the axioms, etc.
+One says the $\sigma$-algebra is \emph{generated} by these sets.
+Indeed, \verb|Distributions.jl| does not give means to compute probability for arbitrary subsets of floating point numbers,
+but gives a way of computing probability mass just of intervals of the form $(-\infty, x]$ through the function \verb|cdf|.
+
+For spaces that are ``small enough'', a meaningful probability can be assigned to each subset of the space. Unfortunately, this is impossible for any space containing a continuum such as the real line. Just representing a single arbitrary subset of the \verb|Float64|-range, the computational abstraction of the real line, would require staggering 2\,306 Petabytes of memory. This is not practical.
```

Comment:
```suggestion
For spaces that are ``small enough'', a meaningful probability can be assigned to each subset of the space. 
But even in moderately sized spaces this can become infeasible: Just representing a single arbitrary subset of the \verb|Float64|-range, the computational abstraction of the real line, would require staggering 2\,306 Petabytes of memory. This is not practical. And for any space containing a continuum such as the real line it becomes not impractical but mathematically impossible to assign a meaningful probability to all subsets. 
```